src/DynamoCore/DynamoCore.csproj

```diff
@@ -176,9 +176,11 @@
     <ItemGroup>
       <ASMLibG Include="$(SolutionDir)..\extern\LibG_*\*" />
       <ExternProtoGeometry Include="$(SolutionDir)..\extern\ProtoGeometry\*" />
+	  <WebView2Libs Include="$(SolutionDir)..\extern\WebView2\*\*\*" />
```

Comment:
Is there a way to copy them from the package folder?

Reply:
do you mean to copy the two dll files from "C:\Users\my_user\.nuget\packages\microsoft.web.webview2\1.0.1264.42" to "Dynamo\bin\AnyCPU\Debug\runtimes" folder, right?

Reply:
yeah, just curious.. But I guess this way would also work, just that we had to keep the hard copy before our next Global launch? @reddyashish @zeusongit

Reply:
@QilongTang @RobertGlobant20 look at my latest PR:
https://github.com/DynamoDS/Dynamo/pull/13949

this will show you how to at least partially copy the file from the package while avoiding hard coding the base path - of course the path will likely depending on target. (net 48, net 6 etc - so you may need some conditional logic) or to find the correct magical variable..

Reply:
Thanks @mjkkirschner That's what I would like to suggest @RobertGlobant20 to follow

Reply:
Fixed in the next commit:
https://github.com/DynamoDS/Dynamo/pull/13947/commits/167528f428e281db3da328f4fb324ce00092701b